sdf.md

```diff
@@ -1841,28 +1856,109 @@ Specification:
+| items                | items of an array                                                 |
+| label                | short text (no constraints); defaults to key                      |
+| maxItems             | maximum number of items in an array                               |
+| maxLength            | maximum length (in characters) for a text string                  |
+| maximum              | maximum for a number                                              |
+| minItems             | minimum number of items in an array                               |
+| minLength            | minimum length (in characters) for a text string                  |
+| minimum              | minimum for a number                                              |
+| multipleOf           | step size of number                                               |
+| nullable             | boolean: can the item be left out?                                |
+| observable           | boolean: can the item be observed?                                |
+| pattern              | regular expression pattern for a text string                      |
+| properties           | named dataqualities for type="object"                             |
+| readable             | boolean: can the item be read?                                    |
+| required             | which data items are required?                                    |
+| sdfChoice            | named dataqualities for a choice                                  |
+| sdfData              | named dataqualities for an independent data type definition       |
+| sdfInputData         | input data to an action                                           |
+| sdfOutputData        | output data of an action or event (sdfRequired applies here)      |
+| sdfRef               | sdf-pointer to definition being referenced                        |
+| sdfRequired          | pointer-list to declarations of required components               |
+| sdfRequiredInputData | pointer-list to declarations of required input data for an action |
+| sdfType              | more detailed information about the type of a string              |
+| type                 | general category of data type                                     |
+| uniqueItems          | boolean: do the items of the array need to be all different?      |
+| unit                 | engineering unit and scale (per SenML registry)                   |
+| writable             | boolean: can the item be written to?                              |
+{: #tab-quality-names title="Initial Content of Quality Names Registry"}
 
 ### Quality Name Prefixes {#qnp}
 
-IANA is requested to create a "Quality Name Prefixes" sub-registry in
-the "SDF Parameters" registry, with the following template:
+IANA is requested to create a "Quality Name Prefixes" registry in
+the "SDF Parameters" registry group, with the following template:
 
 Prefix:
 : A name composed of lower case ASCII letters and digits, starting
```

Comment:
If the description suggestion above for quality name is adopted, should update also here.